src/magicgui/backends/_ipynb/widgets.py

```diff
@@ -248,7 +250,7 @@ def _mgui_get_current_choice(self) -> str:
 
     def _mgui_set_choice(self, choice_name: str, data: Any) -> None:
         """Set the data associated with a choice."""
-        self._ipywidget.options = (*self._ipywidget.options, (choice_name, data))
+        pass
```

Comment:
this change looks suspicious.  why are we removing the ability to set a single choice?  Won't this change mess up calling `CategoricalWidget.set_choice`?

Reply:
You're absolutely right. I may have gotten hung up on the protocol's definition of choice_name update. While my commit seemed to function without breaking tests, this change was completely unnecessary. I'm changing it later.